src/main/java/inventory/mysql/ElasticSearch.java

```diff
@@ -0,0 +1,94 @@
+package inventory.mysql;
```

Comment:
This file will get all the rows from database and update the elasticsearch cache